hadoop-ozone/dist/src/main/compose/ozonesecure/docker-config

```diff
@@ -35,6 +35,7 @@ OZONE-SITE.XML_hdds.grpc.tls.enabled=true
 OZONE-SITE.XML_ozone.replication=3
 OZONE-SITE.XML_ozone.datanode.pipeline.limit=1
 OZONE-SITE.XML_hdds.scmclient.max.retry.timeout=30s
+OZONE-SITE.XML_ozone.scm.ratis.enable=false
```

Comment:
Unsure if we really need these here, what is the reason to disable ratis for this docker env and for the non-HA one?

Reply:
Right, this ozonesecure directory is supposed for the non-HA case.  There is another ozonesecure-ha directory for the HA case.